ecosystem/platform/server/app/controllers/settings_controller.rb

```diff
@@ -27,19 +27,15 @@ def profile_update
 
   def connections
     store_location_for(current_user, request.path)
-    @authorizations = @user.authorizations.to_h do |authorization|
-      [authorization.provider, authorization]
-    end
+    @authorizations = @user.authorizations.group_by(&:provider)
   end
 
   def connections_delete
     store_location_for(current_user, request.path)
-    @authorizations = @user.authorizations.to_h do |authorization|
-      [authorization.provider, authorization]
-    end
+    @authorizations = @user.authorizations
```

Comment:
Should still be group_by as this method uses the connections.html.erb template, which expects authorizations to be a hash.